2025/GeoJSON/street_lines.geojson

```diff
@@ -0,0 +1 @@
d":590,"geometry":{"type":"LineString","coordinates":[[-119.21045091781059,40.800619570679032],[-119.21007803650377,40.800733778829738],[-119.20967685596531,40.800848907489581],[-119.20927243177613,40.800957286259177],[-119.208864960602,40.801058862431887],[-119.20845464059278,40.801153586608848],[-119.20804167128671,40.801241412723414]]},"properties":{"FID":590,"name":"Ishiguro","width":"30","type":"arc"}},{"type":"Feature","id":591,"geometry":{"type":"LineString","coordinates":[[-119.20777983889309,40.80049901792853],[-119.20804167128671,40.801241412723414]]},"properties":{"FID":591,"name":"10:00","width":"40","type":"radial"}},{"type":"Feature","id":592,"geometry":{"type":"LineString","coordinates":[[-119.21340081909409,40.800643261274502],[-119.2132814777184,40.800696118232395],[-119.21289494514053,40.800859047574761],[-119.2125039395321,40.801015693690431],[-119.21210863774462,40.801165985716523],[-119.21170921857669,40.801309855664499],[-119.21130586269305,40.801447238449782],[-119.21096236633156,40.801557628284343]]},"properties":{"FID":592,"name":"Kilgore","width":"50","type":"arc"}},{"type":"Feature","id":593,"geometry":{"type":"LineString","coordinates":[[-119.21069984378084,40.801076134210405],[-119.21069989865754,40.801076234861625],[-119.21096236633156,40.801557628284343]]},"properties":{"FID":593,"name":"9:45","width":"40","type":"radial"}},{"type":"Feature","id":594,"geometry":{"type":"LineString","coordinates":[[-119.21096236633156,40.801557628284343],[-119.21096260353647,40.801558063343066]]},"properties":{"FID":594,"name":"9:45","width":"40","type":"radial"}},{"type":"Feature","id":595,"geometry":{"type":"LineString","coordinates":[[-119.21069984378084,40.801076134210405],[-119.21068730109027,40.801080197296415],[-119.21028234407146,40.801203556022699],[-119.2098739580191,40.801320205958632],[-119.20946233540036,40.801430092124008],[-119.20904767021068,40.801533162726571],[-119.20863015788245,40.801629369185378],[-119.2082099951927,40.801718666155523]]},"properties":{"FID":595,"name":"Jemison","width":"30","type":"arc"}},{"type":"Feature","id":596,"geometry":{"type":"LineString","coordinates":[[-119.20804167128671,40.801241412723414],[-119.2082099951927,40.801718666155523]]},"properties":{"FID":596,"name":"10:00","width":"40","type":"radial"}},{"type":"Feature","id":597,"geometry":{"type":"LineString","coordinates":[[-119.21096236633156,40.801557628284343],[-119.2108987525429,40.801578071923302],[-119.2104880722772,40.801702296897247],[-119.21007400766518,40.801819857173768],[-119.2096567460099,40.801930699569482],[-119.20923647606412,40.802034773940136],[-119.20881338794396,40.802132033202781],[-119.20838767304309,40.802222433357713]]},"properties":{"FID":597,"name":"Kilgore","width":"50","type":"arc"}},{"type":"Feature","id":598,"geometry":{"type":"LineString","coordinates":[[-119.2082099951927,40.801718666155523],[-119.20838767304309,40.802222433357713]]},"properties":{"FID":598,"name":"10:00","width":"40","type":"radial"}}]}
```

Comment:
`"id":333`, and `"id":246` look to be removable as they are creating duplicate roads under 1 ft from existing radial roads. They come from the arc around center camp crossing 5:30 & 6:30 before Atwood. Alternatively the arc could meet 5:30 & 6:30 exactly at Atwood.

https://github.com/user-attachments/assets/d25cd12c-d368-4008-af94-1b542b645d9e

Reply:
@justinormont can you confirm that these issues are not present in the 2024 geojson? We are very limited in resources for making these exports this year and I hesitate to make changes like these directly because i'm not the one making the exports. But if we can confirm that these look like real issue we can go from there.

Reply:
Ok. I'll merge this. We can revisit if there are major issues.